Sources/FluidAudio/ASR/TDT/BlasIndex.swift

```diff
@@ -0,0 +1,15 @@
+import Accelerate
+
+#if ACCELERATE_NEW_LAPACK
+typealias BlasIndex = __LAPACK_int
+#else
+typealias BlasIndex = Int32
+#endif
+
+@inline(__always)
```

Comment:
the accelerate flag only works for us to stop seeing the warning when building, still a problem when people don't set the flags in xcode